tests/orderly/examples/mp/mp.py

```diff
@@ -0,0 +1,24 @@
+import multiprocessing
+import random
+
+import orderly
+
+# ruff: noqa: F821
+
+
+def square(x):
+    return x * x
+
+
+if __name__ == "__main__":
+    orderly.parameters(method=None)
+    orderly.artefact("Squared numbers", "result.txt")
+
+    mp = multiprocessing.get_context(method)  # type: ignore
+    data = [random.random() for _ in range(10)]  # noqa: S311
```

Comment:
I think we can globally exclude this check?

Reply:
Done.